export_faq.md

```diff
@@ -0,0 +1,55 @@
+## Export FAQ
+
+### What happens if I export an HDF5 file to a new HDF5 file with soft or external links?
```

Comment:
For development this is fine, but for user-docs I would suggest a slightly different approach for this  FAQ. Specificaly I would suggest to: 

1. Split the question into "internal links (i.e., soft links)" and "links to other files (i.e., external links)". I believe the answer for SoftLinks is imply that they will be preserved in all cases. It's only for external links where things get a tricky.
2. For external links, rather than breaking things up right away into "Situations", I would suggest to divide this into two main cases: 1) preserve external links and 2) resolve external links (i.e., copy the data) and then list the different cases there.
3. Instead of just numbering the different cases it would be nice to give them names
3.

Reply:
I updated this in `docs/source/export.rst`